src/platform/ESP32/PlatformManagerImpl.cpp

```diff
@@ -118,6 +118,24 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
     return chip::DeviceLayer::Internal::ESP32Utils::MapError(err);
 }
 
+CHIP_ERROR PlatformManagerImpl::_GetCurrentHeapFree(uint64_t & currentHeapFree)
+{
+    currentHeapFree = heap_caps_get_free_size(MALLOC_CAP_8BIT);
```

Comment:
`MALLOC_CAP_DEFAULT` for now. Better use `esp_get_free_heap_size()`

Reply:
Done